apps/frontend/src/app/5_pages/AavePage/components/BorrowAssetsList/components/BorrowModal/BorrowForm.tsx

```diff
@@ -35,81 +35,73 @@ type BorrowFormProps = {
   onSuccess: () => void;
 };
 
-export const BorrowForm: FC<BorrowFormProps> = ({ asset }) => {
-  const reserves = useAaveReservesData();
+export const BorrowForm: FC<BorrowFormProps> = ({ asset, onSuccess }) => {
   const userReservesSummary = useAaveUserReservesData();
   const [borrowAsset, setBorrowAsset] = useState<string>(asset);
   const [borrowAmount, setBorrowAmount, borrowSize] = useDecimalAmountInput('');
   const [acknowledge, setAcknowledge] = useState<boolean>(false);
   const { handleBorrow } = useAaveBorrow();
 
-  const reserve = useMemo(() => {
-    return reserves.find(r => r.symbol === borrowAsset);
-  }, [reserves, borrowAsset]);
+  const borrowableAssetsOptions = useMemo(
+    () =>
+      userReservesSummary.reserves
+        .filter(r => r.reserve.borrowingEnabled)
+        .map(r => ({
+          value: r.reserve.symbol,
+          label: (
+            <AssetRenderer
+              showAssetLogo
+              asset={r.reserve.symbol}
+              chainId={BOB_CHAIN_ID}
+              assetClassName="font-medium"
+            />
+          ),
+        })),
+    [userReservesSummary.reserves],
+  );
```

Comment:
filter + map should be replace by a reduce

Reply:
Perfect, I'll push in separate pr. Thanks!